### PR TITLE
fix bug in rating

### PR DESCRIPTION
**Pull request by matikrk**

### CODE REVIEW

File: src/docs/index2.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (vergilius) [inline src/docs/index2.html]: aw?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "style-guide",
-  "version": "111.0.1",
+  "version": "111.0.3",
```
Comment (aju) [inline package.json]: :O

Comment (matikrk) [inline package.json]: some problems on github. now 111.0.2

Comment (aju) [inline package.json]: change it to 111.0.2 :)

**Pull request merged**